### PR TITLE
revised index and contribute

### PR DESCRIPTION
**Pull request by turbomarc**
Please consider these revisions to index.html and contribute.md.

Key point regarding the "Bite! Clojure" idiom. I _think_ you're basically going for a "Sink your teeth into…" or "Dive into…" phrase, though I went with a simple "Get started…" word choice. The challenge with the current "Bite! Clojure" idiom is that many users will translate this negatively, as in  http://english.stackexchange.com/questions/27753/meaning-and-usage-of-bite-me:
"Bite me is typically used as a US idiomatic expression of discontent or annoyance with another party"

or http://www.urbandictionary.com/define.php?term=bites:
To rip off another person's style, especially with respect to music or fashion.

or http://phils-english.blogspot.com/2010/03/man-this-bites-part-2.html:
A: Man, I have no money to go to the party.
B: Dude, that bites!

A:The food in my school cafeteria bites!
meaning,
B: The food in my school cafeteria sucks

Basically, there is no positive connotation of any "Bite!" phraseology.

Thanks for considering.

### CODE REVIEW

**Comment by yokolet** [comment]
Thanks!!

I meant "Bite! Clojure (cookie)." But, it looks not sounds good for English speakers. I agree with the change. Thanks for explaining it.
